<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>JpegOptionsPanel</class>
  <widget class="QDialog" name="JpegOptionsPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>306</width>
        <height>151</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Jpeg Options</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>4</number>
      </property>
      <item>
        <layout class="QGridLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item row="0" column="0">
            <widget class="QLabel" name="qualityLabel">
              <property name="text">
                <string>Quality:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="colorSubsamplingLabel">
              <property name="text">
                <string>Color sub-sampling:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QCheckBox" name="progressiveCheckBox">
              <property name="text">
                <string>Progressive</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QComboBox" name="colorSubsamplingBox">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <item>
                <property name="text">
                  <string>None - Best Quality (4:4:4)</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>2x1 - Medium Quality (4:2:2)</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>2x2 - Worst Quality (4:1:0)</string>
                </property>
              </item>
            </widget>
          </item>
          <item row="0" column="1">
            <layout class="QHBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>4</number>
              </property>
              <item>
                <widget class="QSlider" name="qualitySlider">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>0</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>75</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="maximumSize">
                    <size>
                      <width>100</width>
                      <height>32767</height>
                    </size>
                  </property>
                  <property name="minimum">
                    <number>1</number>
                  </property>
                  <property name="maximum">
                    <number>100</number>
                  </property>
                  <property name="pageStep">
                    <number>10</number>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                  </property>
                  <property name="tickInterval">
                    <number>10</number>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="qualityValueLabel">
                  <property name="frameShadow">
                    <enum>QFrame::Plain</enum>
                  </property>
                  <property name="text">
                    <string>85</string>
                  </property>
                  <property name="textFormat">
                    <enum>Qt::AutoText</enum>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignVCenter|Qt::AlignLeft</set>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="Spacer21">
                  <property name="sizeHint">
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QVBoxLayout">
          <property name="margin">
            <number>11</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item>
            <spacer name="Spacer10_2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string>Esc</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="4" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
